Don't update XDG mimetypes during build. This is done by the ports framework.

--- kmymoney/CMakeLists.txt.orig	2018-05-28 07:59:03 UTC
+++ kmymoney/CMakeLists.txt
@@ -111,7 +111,6 @@ install(FILES org.kde.kmymoney.appdata.xml
 install(FILES x-kmymoney.xml
   DESTINATION ${XDG_MIME_INSTALL_DIR})
 
-update_xdg_mimetypes(${XDG_MIME_INSTALL_DIR})
 
 ########## Application Icons #############
 install(DIRECTORY icons/hicolor
